libs/core/src/components/pds-textarea/pds-textarea.tsx

```diff
@@ -23,66 +23,66 @@ export class PdsTextarea {
   @Prop() componentId!: string;
 
   /**
-   * Indicates whether or not the textarea is disabled
+   * Indicates whether or not the textarea is disabled.
    * @defaultValue false
    */
   @Prop() disabled = false;
 
   /**
-   * Specifies the error text and provides an error-themed treatment to the field
+   * Specifies the error message and provides an error-themed treatment to the field.
```

Comment:
I believe the invalid prop provides the "theme" not the errorMessage.

Reply:
This has been updated.